sql/src/main/java/io/crate/execution/support/RepositoryService.java

```diff
@@ -20,7 +20,7 @@
  * agreement.
  */
 
-package io.crate.executor.transport;
+package io.crate.execution.support;
```

Comment:
I think this should go into `io.crate.execution.ddl`

Reply:
Yep you're right, for some reason I thought it's also used to query `sys.repositories`